packages/console/src/pages/ApplicationDetails/index.tsx

```diff
@@ -160,18 +86,27 @@ const ApplicationDetails = () => {
                 {t('application_details.advanced_settings')}
               </TabNavLink>
             </TabNav>
-            <form className={styles.form} onSubmit={onSubmit}>
-              <div className={styles.fields}>
-                {isAdvancedSettings ? AdvancedSettingsPage : SettingsPage}
-              </div>
-              <div className={styles.submit}>
-                <Button
-                  htmlType="submit"
-                  type="primary"
-                  title="admin_console.application_details.save_changes"
-                />
-              </div>
-            </form>
+            <FormProvider {...formMethods}>
+              <form className={styles.form} onSubmit={onSubmit}>
+                <div className={styles.fields}>
+                  {isAdvancedSettings ? (
+                    <AdvancedSettingsForm
+                      tokenEndpoint={oidcConfig.token_endpoint}
```

Comment:
Can we put oidcConfig to some context, so no need to pass in the value to every component?

Reply:
These values may need to be included in the application data.
We have a discussion on this. (see: LOG-1908)